Sources/LiftNavigationController.swift

```diff
@@ -2,7 +2,7 @@ import UIKit
 
 open class LiftNavigationController: UIViewController {
     public static let switchAnimationDuration = 0.2
-    public static let navigationBarHeight = CGFloat(64.0)
+    public static let navigationBarHeight = CGFloat(90.0)
```

Comment:
yup, but this is the weird thing for the navigationbar height that doesnt match the actual designs cause its impossible